lib/sass/calcite-web/base/_colors.scss

```diff
@@ -167,11 +167,11 @@ $darkest-gray:                $Calcite_Gray_600;
 $off-black:                   $Calcite_Gray_650;
 $black:                       $Calcite_Gray_700;
 
-$transparent-white:           rgba(255, 255, 255, 0.7);
```

Comment:
Im kind of a fan of the leading zeroes before decimal points - helps make it clear that its not an integer